helm/templates/configmap.yaml

```diff
@@ -0,0 +1,9 @@
+{{- $chartName := include "opa.name" . -}}
+{{- if .Values.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-{{ $chartName }}-configmap
+data:
+  OPA_CONFIG: {{ tpl (.Files.Get "config/opa.yaml") . | quote }}
```

Comment:
make path configurable with a default value

Reply:
why do wee need it as a configuration?